packages/pds/src/app-view/api/app/bsky/unspecced.ts

```diff
@@ -43,18 +43,6 @@ export default function (server: Server, ctx: AppContext) {
       }
     },
   })
-
-  server.app.bsky.unspecced.applyLabels({
```

Comment:
I expect we'll want to remove the use of this endpoint in `packages/bsky/src/auto-moderator/index.ts`.

Reply:
yup - wiped out the whole schema 👌